datadog_checks_dev/datadog_checks/dev/tooling/commands/validate/metadata.py

```diff
@@ -293,6 +293,11 @@ def metadata(check):
                             current_check, row['metric_name'], MAX_DESCRIPTION_LENGTH
                         )
                     )
+                if row['interval'] and not isinstance(row['interval'], int):
+                    errors = True
+                    echo_failure(
+                        'interval should be an int, found {}'.format(row['interval'])
```

Comment:
You'd also want to display the `current_check`, otherwise logs are difficult to interprets.